Modules/II-k6-Foundations/09-Recording-a-k6-script.md

```diff
@@ -27,8 +27,8 @@ Then, open up a new tab and click on the k6 browser extension icon. You'll see a
 
 ![k6-browser-recorder-01](../../images/k6-browser-recorder-01.png)
 Here's what those options mean:
-- **Download HAR file**: HAR stands for HTTP ARchive, which is a JSON-formatted file that saves network information like HTTP requests and timings. Enable this if you'd like to be able to be able to save the recording somewhere other than k6. Otherwise, **leave this disabled**.
-- **Clear cache (last 7 days):** Enabling this deletes your browser cache for the last week. This is useful if what you want to simulate a new user to your application. Disable this if you want to simulate an existing user. Otherwise, **leave this option enabled**. (Read more about [caching options here](../XX-Future-Ideas/Caching-options.md).)
+- **Download HAR file**: HAR stands for HTTP ARchive, which is a JSON-formatted file that saves network information like HTTP requests and timings. Enable this if you'd like to be able to save the recording somewhere other than k6. Otherwise, **leave this disabled**.
+- **Clear cache (last 7 days):** Enabling this deletes your browser cache for the last week. This is useful if what you want to simulate a new user to your application. Disable this if you want to simulate an existing user. Otherwise, **leave this option enabled**. Read more about [caching options here](../XX-Future-Ideas/Caching-options.md).
```

Comment:
```suggestion
- **Download HAR file**: HAR stands for HTTP ARchive, which is a JSON-formatted file that saves network information like HTTP requests and timings. **Enable this if you'd like to be able to save the recording somewhere other than k6**. 
- **Clear cache (last 7 days):** Enabling this deletes your browser cache for the last week. This is useful if what you want to simulate a new user to your application. **Disable this if you want to simulate an existing user.** 
-  Read more about [caching options](../XX-Future-Ideas/Caching-options.md).
```

Reply:
Since bold jumps out to the eye, I think we want to bold phrases that convey the most information. "Leave enabled" doesn't say much without the surrounding context.

I know this is outside of this PR's changes anyway, but just to document my reasoning.